### DISCUSSION TITLE
Is it possible to make "Starfield" from Steam work, on Whisky GPTK?

### DISCUSSION THREAD

**Discussion by ferreira-ff** [Whisky-App/Whisky#352] [Q&A] [answered]
As the title; I really wanted to play this game. I am unable to run it… is there anything I can install from Winetricks, to make it work?

Thank you and kind regards to all.

**Comment by IsaacMarovitz** [comment, marked as answer]
Starfield uses AVX instructions. Therefore, it is impossible to run it on GPTK.

**Comment by Kujarkh** [comment]
And now with gptk 2 ? Thanks
